config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-release-1-6-upgrades.yaml

```diff
@@ -0,0 +1,319 @@
+periodics:
+
+- name: periodic-cluster-api-e2e-workload-upgrade-1-22-1-23-release-1-6
```

Comment:
Looks like all the new jobs in the PR are not based on the current main branch jobs as described on the release tasks [notes](https://github.com/kubernetes-sigs/cluster-api/blob/main/docs/release/release-tasks.md#setup-jobs-and-dashboards-for-a-new-release-branch)? Have you copied these jobs based on the current main jobs, i.e [cluster-api-periodics-release-1-6-upgrades.yaml] needs to be created based on the https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-main-upgrades.yaml

Reply:
sorry my bad, now it's fixed.